### DISCUSSION TITLE
Creador de elicoin

### DISCUSSION THREAD

**Discussion by Eliecer221** [community/community#20261] [Other Feature Feedback, Questions, & Ideas]
Buenas la presente es para solicitar una asesoría yo soy el creador de la criptomoneda elicoin me he dado cuenta que tienen correos para contactar a los creadores de elicoin pero a mí no me llega nada y vi que solicitan donación de criptomonedas y Yo no he pedido nada yo la cree con esta aplicación 
console.chainstack.com solicito ayuda y asesoría por favor.

**Comment by Eliecer221** [comment]
Contactarme WhatsApp +1 4135919397

**Comment by diegoleondev** [comment]
>**Warning**
>**Posible Estafa** / **Possible Scam**
